packages/jellyfish-transaction/__tests__/script/index.test.ts

```diff
@@ -720,6 +720,348 @@ describe('[OP_CHECKMULTISIGVERIFY]', () => {
   })
 })
 
+describe('[OP_TOALTSTACK]', () => {
```

Comment:
Move all the code to the line after OP_RETURN.
The hex for OP_RETURN is '016a'
So 016a, 016b, 016c ...